terraform/aks/variables.tf

```diff
@@ -250,3 +250,5 @@ variable azure_tempdata_storage_account_name {}
 variable azure_storage_account_replication_type { default = "LRS"}
 
 variable azure_region_name { default = "uk south" }
+
+variable "deploy_temp_data_storage_account" {default = true }
```

Comment:
`{default = true }` to `{ default = true }` for consistency. Maybe a `terraform fmt` or the VS Code plugin?